errors/errors.go

```diff
@@ -100,3 +101,18 @@ func StackTrace(err error) []string {
 
 	return nil
 }
+
+func Combine(errs []error) error {
+	if len(errs) > 0 {
```

Comment:
👍 I submit PRs to projects with opposing code styles for this case and sometimes I forget 😄